server-common/src/test/java/org/apache/kafka/server/util/MockTime.java

```diff
@@ -27,15 +23,21 @@
  * 1. This has an associated scheduler instance for managing background tasks in a deterministic way.
  * 2. This doesn't support the `auto-tick` functionality as it interacts badly with the current implementation of `MockScheduler`.
  */
-class MockTime(currentTimeMs: Long, currentHiResTimeNs: Long) extends JMockTime(0, currentTimeMs, currentHiResTimeNs) {
-
-  def this() = this(System.currentTimeMillis(), System.nanoTime())
+public class MockTime extends org.apache.kafka.common.utils.MockTime {
```

Comment:
My preference would be to rename this class. Having two MockTime is super confusing.

Reply:
I agree that the name is bad. As said earlier, my intent is to just move it for now. We could consider renaming it separately.